test/integrationConfiguration.test.js

```diff
@@ -0,0 +1,217 @@
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+
+
+        process.env.ASSET_COMPUTE_INTEGRATION_FILE_PATH = jsonFile;
+        process.env.ASSET_COMPUTE_PRIVATE_KEY_FILE_PATH = privateKeyFile;
+        actual = await IntegrationConfiguration.getConfiguration();
+
+        assert.strictEqual(actual.metascopes[0], expected.metascopes[0]);
+        assert.strictEqual(actual.metascopes[1], expected.metascopes[1]);
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+
+
+        process.env.ASSET_COMPUTE_PRIVATE_KEY_FILE_PATH = privateKeyFile;
+        actual = await IntegrationConfiguration.getConfiguration(jsonFile);
+
+        assert.strictEqual(actual.metascopes[0], expected.metascopes[0]);
+        assert.strictEqual(actual.metascopes[1], expected.metascopes[1]);
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+    });
+
+    it('Input file is missing properties', async () => {
+
+        const yamlFile = path.join(process.cwd(), "test", "resources/integration-missing-properties.yaml");
+        const jsonFile = path.join(process.cwd(), "test", "resources/integration-missing-properties.json");
+
+
+        let actual;
+        const expected = new Error(`Not all integration configuration properties are present`);
+
+        try {
+            await IntegrationConfiguration.getConfiguration(jsonFile, privateKeyFile);
+        } catch (e) {
+            actual = e;
+        }
+
+        assert.strictEqual(actual.name, expected.name);
```

Comment:
Put this in its own test section (own `it(...)) to ensure there are no leftovers in `actual` from the test L192 to 203.